src/Niekert/Discord/Main.php

```diff
@@ -18,12 +19,31 @@ public function onEnable(){
 		$username = $this->getConfig()->get("username");
 		$message = $this->getConfig()->get("message");
 		$webhook = $this->getConfig()->get("webhook_url");
-		$this->reloadConfig();
-		$data = array("content" => $message, "username" => "$username",);
-		$curl = curl_init("$webhook");
-		curl_setopt($curl, CURLOPT_CUSTOMREQUEST, "POST");
-		curl_setopt($curl, CURLOPT_POSTFIELDS, json_encode($data));
-		curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
-		return curl_exec($curl);
+                
+                // Statements
+                
+                if ($username === "") {
+                    $this->getLogger()->warning('Please set your username in config.yml');
+                }
+                elseif ($webhook === "") {
+                    $this->getLogger()->warning('Please set your webhook in config.yml');
+                }
+                elseif ($message === "") {
+                    $this->getLogger()->warning('Please set your message in config.yml');
+                }
+                else {
+                
+                    $this->reloadConfig();
+                    $data = array("content" => $message, "username" => "$username");

```

Comment:
If you are reloading config you are suppose to re fetch the variables or there's no point in reloading it